raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -802,6 +806,22 @@ contract TokenNetwork is Utils {
         }
     }
 
+    function getLocksrootIdentifier(
+        bytes32 channel_identifier,
+        address participant,
+        bytes32 locksroot
+    )
+        pure
+        public
+        returns (bytes32 key)
+    {
+        require(channel_identifier != 0x0);
+        require(participant != 0x0);
+        require(locksroot != 0x0);
+
+        key = keccak256(abi.encodePacked(channel_identifier, participant, locksroot));
+    }
```

Comment:
I don't understand,

> the function get's only used once

are you referring to `getLocksrootIdentifier` ? Because this is used 3 times. And I think the checks are also important.

Reply:
At least `require(participant != 0x0);` is done in getChannelIdentifier already. I'd prefer less code here, but your PR.

Reply:
Removed redundant checks in https://github.com/raiden-network/raiden-contracts/pull/160/commits/84586732a5c83476c4efbea6478776478aeaaaef